cmd/deploy.go

```diff
@@ -238,6 +238,20 @@ func (dc *deployCmd) loadAPIModel() error {
 				dc.containerService.Properties.CustomCloudProfile.IdentitySystem = dc.authProvider.getAuthArgs().IdentitySystem
 			}
 		}
+
+		if dc.containerService.Properties.IsAzureStackCloud() {
+			if dc.containerService.Properties.MasterProfile.Distro == api.AKSUbuntu1604 {
+				log.Errorln("Distro 'aks-ubuntu-16.04' is not longer supported on Azure Stack Hub, please update master profile distro in api model to use 'aks-ubuntu-18.04'")
```

Comment:
I think the result of these two lines will be two similar messages in stdout. If that's the case, can you just keep one?